README.rst

```diff
@@ -8,3 +8,121 @@ A GPL-licensed Python library that facilitates functional testing of quipucords.
 .. image:: https://codecov.io/gh/quipucords/camayoc/branch/master/graph/badge.svg
    :target: https://codecov.io/gh/quipucords/camayoc
+
+Installation
+^^^^^^^^^^^^
+
+Camayoc only supports Python 3.5 and python 3.6, so it is recommended that you
+install Camayoc into a virtual environment. There are several tools available
+for managing virtual environments, below is an example of one way.::
+
+    git clone https://github.com/quipucords/camayoc.git
+    python3.6 -m vevn ~/envs/camayoc
+    source ~/envs/camayoc/bin/activate
+    cd camayoc
+    # To install for development, install all dependencies
+    make install-dev
+    # If you only want to run the test suite, you can skip
+    # a few dependencies with the plain "install" target
+    make install
+
+Configuration
+^^^^^^^^^^^^
+
+Camayoc requires a configuration file to be installed to
+``$XDG_CONFIG_HOME/camayoc/config.yaml``. On most systems this will be
+``~/.config/camayoc/config.yaml``.
+
+The config file tells camayoc about the hosts you would like to run tests
+against and the credentials to use with these hosts. It also tells camayoc
+where the ``quipucords`` server you want to test if you are running
+``quipucords`` tests.
+
+Camayoc contains test suites for both quipucords/rho and quipucords/quipucords
+projects. All portions of the test suite expect the config file format.  Any
+changes to the test suite that require changes to the config file format should
+be made with this in mind.
+
+There is an example annotated config file in ``camayoc/example_config.yaml``
+
+Configuring py.test
+^^^^^^^^^^^^^^^^^^
+
+If you are testing machines that use ``https`` but do not have good
+certificates, it can get difficult to read the test results due to warnings
+about insecure connections. If this is a problem for you, an example
+``pytest.ini`` file is provided in the root of the ``camayoc`` repository that
+disables these warnings. Additional configuration is possible through the
+``pytest.ini`` file, these options are documented at
+https://docs.pytest.org/en/latest/reference.html#ini-options-ref
+
+For example, if you always want to run ``py.tests`` in verbose mode, this can
+be done in the ``pytest.ini`` file.
+
+Running the test suite
+^^^^^^^^^^^^
+
+While ``camayoc`` contains test for ``rho`` and ``quipucords``, and these share
+a config file format, testing both ``rho`` and ``quipucords`` simultaneously is
+not a supported use case.
+
+Camayoc uses ``py.test`` as its test driver. The pytest project had excellent
+documentation available at https://docs.pytest.org/en/latest/contents.html
+
+Running tests for ``rho``
+^^^^^
+
+To run all the tests for ``rho``, first have ``rho`` installed and available in
+your ``$PATH``. This can be done by either ``rpm`` install to the system or by
+``pip`` installing ``rho`` from source into your ``camayoc`` virtual
+environment. Then invoke the main test suite from the root directory of
+``camayco`` with::
+
+    py.test camayoc/tests/rho/
```

Comment:
I think there is an opportunity to just add a `make` target for simplicity.

`make test-rho`